tungsten_tempest_plugin/tests/api/contrail/test_routers.py

```diff
@@ -250,59 +224,66 @@ def test_delete_bgp_router(self):
     @rbac_rule_validation.action(service="Contrail",
                                  rules=["list_global_vrouter_configs"])
     @decorators.idempotent_id('4af768d1-3cbe-4aff-bcbc-0e045cac3277')
-    def test_list_global_vrouter(self):
+    def test_list_global_vrouter_configs(self):
         """test method for list global vrouter config objects"""
         with self.rbac_utils.override_role(self):
             self.router_client.list_global_vrouter_configs()
 
+    @decorators.skip_because(bug="1792446")
     @rbac_rule_validation.action(service="Contrail",
                                  rules=["create_global_vrouter_configs"])
     @decorators.idempotent_id('e13d800f-9304-4a06-9bf1-ad08345a13a8')
-    def test_create_global_vrouter(self):
+    def test_create_global_vrouter_configs(self):
         """test method for create global vrouter config objects"""
-        # Create global system config
-        global_system_config = self._create_global_system_config()['name']
+        # This test may make your environment unstable
+        # Juniper JTAC 2018-0912-0503
         with self.rbac_utils.override_role(self):
-            self._create_global_vrouter_config(global_system_config)
+            self._create_global_vrouter_config()
 
+    @decorators.skip_because(bug="1792446")
     @rbac_rule_validation.action(service="Contrail",
                                  rules=["show_global_vrouter_config"])
     @decorators.idempotent_id('3bb6f4e1-fd3f-4338-8392-f7f80974a80e')
     def test_show_global_vrouter_config(self):
         """test method for show global vrouter config objects"""
-        # Create global system config
-        global_system_config = self._create_global_system_config()['name']
-        global_vrouter_config_uuid = self._create_global_vrouter_config(
-            global_system_config)['uuid']
+        # This test may make your environment unstable
+        # Juniper JTAC 2018-0912-0503
+        global_vrouter_config_uuid = \
+            self._create_global_vrouter_config()['uuid']
         with self.rbac_utils.override_role(self):
             self.router_client.show_global_vrouter_config(
                 global_vrouter_config_uuid)
 
+    @decorators.skip_because(bug="1792446")
     @rbac_rule_validation.action(service="Contrail",
                                  rules=["update_global_vrouter_config"])
     @decorators.idempotent_id('36fcdd51-c42b-4e67-8c26-73d4cde47507')
-    def test_update_global_vrouter(self):
+    def test_update_global_vrouter_config(self):
         """test method for update global vrouter config objects"""
-        # Create global system config
-        global_system_config = self._create_global_system_config()['name']
+        # This test may make your environment unstable
+        # Juniper JTAC 2018-0912-0503
```

Comment:
Same comment as above

Reply:
same as above, the test is skipped by decorator: @decorators.skip_because(bug="1792446")